pool/config.go

```diff
@@ -64,4 +64,8 @@ type EffectiveGasPriceCfg struct {
 
 	// FinalDeviationPct is the max allowed deviation percentage BreakEvenGasPrice on re-calculation
 	FinalDeviationPct uint64 `mapstructure:"FinalDeviationPct"`
+
+	// L2GasPriceSuggesterFactor is the factor to apply to L1 gas price to get the suggested L2 gas price used in the
+	// calculations when the effective gas price is disabled (testing/metrics purposes)
+	L2GasPriceSuggesterFactor float64 `mapstructure:"L2GasPriceSuggesterFactor"`
```

Comment:
if this parameter is used only for metrics and testing, is it possible to not be a configuration but a mocked price suggester somewhere that would be injected the same way we do for the real one, but only the situations we need it?

Also, if this parameter is used only when the EGP is disabled, I understand that it means we are always going to have an EGP "enabled", a fixed value and a dinamic value. Not sure if this config is conflicting with the `Enabled` config.

Reply:
It is not used for the tests (CI) it is used in production (internal, testnet, mainnet) when the EGP is disabled to do anyway the calculation and logs the results about what would have happened if the EGP had been enabled. With this info (that is showed in the logs but also store in the state.transaction table) we can do a post analisys to review how the EGP is working an get metrics about the amount of txs executed